spec/lib/jyson/jyson.array.spec.js

```diff
@@ -80,112 +80,110 @@ describe('jyson.array.spec: an array in the template', () => {
-          const input = {
-            a: [
-              {
-                a: 'a0a',
-                c: 'a0c',
-              }, {
-                a: 'a1a',
-                b: 'a1b',
-                c: 'a1c',
-              }
-            ],
-          };
-          const json = this.templateFunction(input);
-          expect(json.a.length).to.equal(2);
-          expect(json.a[0].a).to.equal('a0a');
-          expect(json.a[0].b).to.equal(null);
-          expect(json.a[0].c).to.equal('a0c');
-          expect(json.a[1].a).to.equal('a1a');
-          expect(json.a[1].b).to.equal('a1b');
-          expect(json.a[1].c).to.equal('a1c');
-        });
+      it('must handel undefined values when converting an object to json', () => {
+        const input = {
+          a: [
+            {
+              a: 'a0a',
+              c: 'a0c',
+            }, {
+              a: 'a1a',
+              b: 'a1b',
+              c: 'a1c',
+            }
+          ],
+        };
+        const json = this.templateFunction(input);
+        expect(json.a.length).to.equal(2);
+        expect(json.a[0].a).to.equal('a0a');
+        expect(json.a[0].b).to.equal(null);
+        expect(json.a[0].c).to.equal('a0c');
+        expect(json.a[1].a).to.equal('a1a');
+        expect(json.a[1].b).to.equal('a1b');
+        expect(json.a[1].c).to.equal('a1c');
+      });
 
-        it('must convert an object of length 10 to json', () => {
-          const input = {
-            a: [
-              {
-                a: 'a0a',
-                b: 'a0b',
-                c: 'a0c',
-              }, {
-                a: 'a1a',
-                b: 'a1b',
-                c: 'a1c',
-              }, {
-                a: 'a2a',
-                b: 'a2b',
-                c: 'a2c',
-              }, {
-                a: 'a3a',
-                b: 'a3b',
-                c: 'a3c',
-              }, {
-                a: 'a4a',
-                b: 'a4b',
-                c: 'a4c',
-              }, {
-                a: 'a5a',
-                b: 'a5b',
-                c: 'a5c',
-              }, {
-                a: 'a6a',
-                b: 'a6b',
-                c: 'a6c',
-              }, {
-                a: 'a7a',
-                b: 'a7b',
-                c: 'a7c',
-              }, {
-                a: 'a8a',
-                b: 'a8b',
-                c: 'a8c',
-              }, {
-                a: 'a9a',
-                b: 'a9b',
-                c: 'a9c',
-              }
-            ],
-          };
-          const json = this.templateFunction(input);
-          expect(json.a.length).to.equal(10);
-          for(let ii = 0; ii < 10; ii++) {
-            expect(json.a[ii].a).to.equal(`a${ii}a`);
-            expect(json.a[ii].b).to.equal(`a${ii}b`);
-            expect(json.a[ii].c).to.equal(`a${ii}c`);
-          }
-        });
+      it('must convert an object of length 10 to json', () => {
```

Comment:
Why specifically length 10?

Reply:
no reason its just a very large test, should it be named better?